web/pages/settings/index.vue

```diff
@@ -66,7 +66,7 @@ function uploadFlow() {
 	flowsStore.uploadFlow(file).then((res: any) => {
 		console.debug('uploadFlow', res)
 		const toast = useToast()
-		if ('error' in res && res?.error !== '') {
+		if (res && 'error' in res && res?.error !== '') {
```

Comment:
as I know we now have not `error` but `detail` and this should be renamed everywhere?